### ISSUE TITLE
`--debug` flag needed to display `API Rate Limit Reached. Waiting 20 seconds` error

### ISSUE THREAD

**Issue by Artemonim** [Significant-Gravitas/Auto-GPT#1441]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

I may have run out of free trial tokens.

### Current behavior 😯

I was getting an error on startup. Only after enabling debug mode did it become clear what was going on.
---
![2023-04-15_02-18-34](https://user-images.githubusercontent.com/36363466/232170409-fc4588d8-2cea-4bcd-a007-a52f9d6c2025.png)
The red zone is important for me
The yellow zone is `--debug` zone and I can't see it without `--debug`
The green zone is normal feedback from the system

### Expected behavior 🤔

I should see these messages even without debug mode

### Your prompt 📝

```yaml
# Paste your prompt here
```

**Comment by ASVN2** [comment]
Same :arrow_up_small:

**Comment by anirbanmajumder0** [comment]
https://platform.openai.com/account/billing/overview
Enter Set up paid account

**Comment by anirbanmajumder0** [comment]
!close

**Comment by Artemonim** [comment]
> https://platform.openai.com/account/billing/overview Enter Set up paid account

I'm not talking about running out of tokens, but about having to run debug to find out.

**Comment by Hypertext-Assassin-RSS** [comment]
Same to me 😒

**Comment by anonhostpi** [comment]
@Artemonim recommend changing title a bit to emphasize the `--debug` flag, because as it stands, I almost closed the issue as a duplicate of the various rate-limit reached error reports we get:
- [Gist Tracker - Rate Limits][gist]
- [Alt Tracker - Rate Limits][source] 
- [Cost Label][label_cost]

[gist]:https://gist.github.com/anonhostpi/97d4bb3e9535c92b8173fae704b76264#file-_topics-0005-api-0002-access-rate-limits-md
[source]:https://github.com/anonhostpi/AUTOGPT.TRACKERS/blob/main/TOPICS/0005.API/0002.ACCESS/RATE-LIMITS.md
[label_chunking]:https://github.com/Significant-Gravitas/Auto-GPT/labels/function%3A%20process%20text
[label_cost]:https://github.com/Significant-Gravitas/Auto-GPT/labels/API%20costs

**Comment by Artemonim** [comment]
> @Artemonim recommend changing title a bit to emphasize the `--debug` flag, because I almost closed the issue as a duplicate of the various rate-limit reached error reports we get

done?

P.S. It is strange that watchers do not have the right to change the header
